code/game/machinery/computer/arcade/arcade.dm

```diff
@@ -67,7 +67,11 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		/obj/item/toy/foamfinger = 2,
 		/obj/item/clothing/glasses/trickblindfold = 2,
 		/obj/item/clothing/mask/party_horn = 2,
-		/obj/item/storage/box/party_poppers = 2))
+		/obj/item/storage/box/party_poppers = 2,
+		//Orbstation additions
+		/obj/item/toy/plush/amoungplushie = 2,
+		/obj/item/toy/plush/tbhplushie = 2
+		))
```

Comment:
```suggestion
		/obj/item/toy/plush/tbhplushie = 2,
	))
```
Leaving a trailing comma at the end means it will play nicer with future changes to the list.